com.playeveryware.eos/Runtime/Core/EOSManager.cs

```diff
@@ -138,6 +135,48 @@ public delegate void OnAuthLinkExternalAccountCallback(
         //cached log levels for retrieving later
         private static Dictionary<LogCategory, LogLevel> logLevels;
 
+        /// <summary>
+        /// This stores a reference to an existing EOSManager.
+        /// This should only be used to determine if a new copy of EOSManager should become not enabled.
+        /// </summary>
+        private static EOSManager existingManager = null;
```

Comment:
As a `private static` field member, this should be prefixed with `s_` to indicate that it is static, and that it is private. Other field members within this file only do not follow this convention because their existence predates that convention, and a task was never defined to retro-actively apply the convention.

Reply:
Noted, will follow this convention.